test/quark-core/Noncer.t.sol

```diff
@@ -357,21 +357,26 @@ contract NoncerTest is Test {
         assertEq(replayCount, 2);
     }
 
-    function testGetActiveReplayCountWithCancel() public {
+    function testGetActiveReplayCountWithNonReplayCancel() public {
```

Comment:
This test has nothing to do with cancellations anymore right? If so, the test name should change since the `Cancel` in it is a bit confusing

Reply:
Yeah, I think this was about a "nop" non-replayable script, which we also call "cancel," so I think it still applies. Took me a minute to realize this, too.